lib/puppet/provider/katello_ssl_tool.rb

```diff
@@ -81,9 +83,8 @@ def deploy!
 
     def new_version_available?
```

Comment:
I don't think this function covers what it needs to do anymore. Something like ```needs_deploy?``` feels more appropriate now.